test/puppetlabs/puppetdb/testutils/events.clj

```diff
@@ -11,24 +11,10 @@
 ;; can be compared for testing
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defn munge-example-event-for-storage
-  "Helper function to munge our example reports into a format suitable for submission
-  via the 'store report' command."
-  [example-event]
-  ;; Because we want to compare 'certname' in the output of event queries, the
-  ;; example data includes it... but it is not a legal key for an event during
-  ;; report submission.
-  (dissoc example-event :certname :containing_class :environment))
-
-(defn environment [resource-event report version]
-  (if (= :v4 version)
-    (assoc-when resource-event :environment (:environment report))
-    (dissoc resource-event :environment)))
-
 (defn expected-resource-event
   "Given a resource event from the example data, plus a report hash, coerce the
   event into the format that we expect to be returned from a real query."
-  [version example-resource-event report]
+  [example-resource-event report]
```

Comment:
meh thinking about it more this probably won't happen for some time. removing it seems fine.